zad5/app/helpers/artists_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ArtistsHelper
```

Comment:
railsy tworzą wiele plików przy generowaniu kontrolerów.
jak się jakiegoś nie używa to można spokojnie skasować 🚜